### PR TITLE
doc: Style cleanup in hypercall hld

### PR DESCRIPTION
**Pull request by amyreye**
Style changes per Acrolinx recommendations and for consistency

Signed-off-by: Reyes, Amy <amy.reyes@intel.com>

### CODE REVIEW

**Comment by acrnsi** [comment]
Can one of the admins verify this patch?

File: doc/developer-guides/hld/hv-hypercall.rst [hunk 1]
```diff
@@ -3,39 +3,40 @@
...
 ######################
 
-The hypercall/upcall is used to request services between the Guest VM and the hypervisor.
-The hypercall is from the Guest VM to hypervisor, and the upcall is from the hypervisor to the Guest VM.
-The hypervisor currently supports hypercall APIs for VM management, I/O request
-distribution, interrupt injection, PCI assignment, guest memory mapping,
-power management, and secure world switch.
+The hypercall/upcall is used to request services between the guest VM and the
+hypervisor. The hypercall is from the guest VM to the hypervisor, and the upcall
+is from the hypervisor to the guest VM. The hypervisor currently supports
```
Comment (dbkinder) [inline doc/developer-guides/hld/hv-hypercall.rst:8]: Let's avoid using "currently" or implying that a future release will do something different:
```suggestion
is from the hypervisor to the guest VM. The hypervisor supports
```

File: doc/developer-guides/hld/hv-hypercall.rst [hunk 2]
```diff
@@ -3,39 +3,40 @@
...
-IPI vector for HV notification to the Service VM. This upcall is necessary whenever
-there is device emulation requirement to the Service VM. The upcall vector (0xF3) is
-injected to Service VM vCPU0. The Service VM will register the IRQ handler for vector (0xF3) and notify the I/O emulation
-module in the Service VM once the IRQ is triggered.
-View the detailed upcall process at :ref:`ipi-management`
+The hypervisor and Service VM both use the same vector (0xF3), which is reserved
+as the x86 platform IPI vector for hypervisor notification to the Service VM.
+This upcall is necessary whenever there is a device emulation requirement to the
+Service VM. The upcall vector (0xF3) is injected to Service VM vCPU0. The
+Service VM will register the IRQ handler for vector (0xF3) and notify the I/O
```
Comment (dbkinder) [inline doc/developer-guides/hld/hv-hypercall.rst:34]: You've been pretty consistent using or changing to use present tense
```suggestion
Service VM registers the IRQ handler for vector (0xF3) and notifies the I/O
```

**Review by dbkinder — CHANGES_REQUESTED** [review]

**Review by dbkinder — APPROVED** [review]
LGTM, thanks!

**Pull request merged**